metrics/grafana/pd.json

```
@@ -6394,19 +7175,29 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "pd_server_etcd_state{type=\"committedIndex\"}",
+              "expr": "histogram_quantile(0.99, sum(rate(pd_server_handle_tso_duration_seconds_bucket[30s])) by (type, le))",
               "format": "time_series",
+              "hide": false,
               "intervalFactor": 2,
-              "legendFormat": "{{instance}} - {{job}}",
-              "refId": "A"
+              "legendFormat": "99%  tso",
+              "refId": "A",
+              "step": 2
+            },
+            {
+              "expr": "histogram_quantile(0.99999, sum(rate(pd_server_handle_tso_duration_seconds_bucket[30s])) by (type, le))",
+              "format": "time_series",
+              "intervalFactor": 1,
+              "legendFormat": "99.999%  tso",
+              "refId": "B"
             }
           ],
           "thresholds": [],
           "timeFrom": null,
           "timeRegions": [],
           "timeShift": null,
-          "title": "Raft committed index",
+          "title": "PD Server TSO  handle time + Client Revc time",
```

```suggestion
          "title": "PD server TSO handle time + Client recv time",
```
BTW, we need to change the description of this panel